common/src/test/kotlin/streams/utils/SchemaUtilsTest.kt

```diff
@@ -20,6 +20,57 @@ class SchemaUtilsTest {
         assertEquals(setOf("foo"), keys)
     }
 
+    @Test
+    fun `getNodeKeys should return the key sorted properly`() {
+        // the method getNodeKeys should select (with multiple labels) the constraint with lowest properties
+        // with the same size, we take the first label in alphabetical order
+        // finally, with same label name, we take the first sorted properties list alphabetically
+
+        val pair1 = "LabelX" to setOf("foo", "aaa")
+        val pair2 = "LabelB" to setOf("bar", "foo")
+        val pair3 = "LabelC" to setOf("baz", "bar")
+        val pair4 = "LabelB" to setOf("bar", "bez")
+        val pair5 = "LabelA" to setOf("bar", "baa", "xcv")
+        val pair6 = "LabelC" to setOf("aaa", "baa", "xcz")
+        val pair7 = "LabelA" to setOf("foo", "aac")
+        val pair8 = "LabelA" to setOf("foo", "aab")
+        val props = mutableListOf(pair1, pair2, pair3, pair4, pair5, pair6, pair7, pair8)
```

Comment:
trasformalo in mappa per favore

Reply:
Ora ricordo perché non l'ho messo come mappa. Se faccio una mappa poi quando faccio `val constraints = props.map {...` , essendo una mappa mi rimuove le chiavi duplicate, quindi rimane solo una "LabelA" e non posso testare l'ordinamento in base al nome delle label.